OpenRA.Mods.Common/Widgets/ListLayout.cs

```diff
@@ -21,22 +21,24 @@ public class ListLayout : ILayout
 		public void AdjustChild(Widget w)
 		{
 			if (widget.Children.Count == 0)
-				widget.ContentHeight = widget.ItemSpacing;
+				widget.ContentHeight = 2 * widget.TopBottomSpacing - widget.ItemSpacing;
```

Comment:
I now see that this is a hacky workaround for the line below (so that the final y is indeed `widget.TopBottomSpacing`).